verify.py

```diff
@@ -0,0 +1,83 @@
+#Importing libraries

```

Comment:
Your Pull Request title: VERIFY-1022 Added verify.py

Check result:

VERIFICATION

* Pull Request title must start with prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']